content/blog/readr-1-4-0/index.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+output: hugodown::hugo_document
+
+slug: readr-1-4-0
+title: readr 1.4.0
+date: 2020-09-30
+author: Jim Hester
+description: >
+    A 2-3 sentence description of the post that appears on the articles page.
+    This can be omitted if it would just recapitulate the title.
+photo:
+  url: https://unsplash.com/photos/XOW1WqrWNKg
+  author: Anastasia Zhenina
+categories: [package]
+tags:
+  - readr
+  - tidyverse
+---
+
+[readr](http://readr.tidyverse.org) 1.4.0 is now available on CRAN! Learn more
+about readr at <https://readr.tidyverse.org>. Detailed notes are always in the
+[change log](https://readr.tidyverse.org/news/index.html#readr-1-4-0).
+
+The readr package makes it easy to get rectangular data out of comma separated
+(csv), tab separated (tsv) or fixed width files (fwf) and into R. It
+is designed to flexibly parse many types of data found in the wild, while still
+cleanly failing when data unexpectedly changes. If you are new to readr, the
+best place to start is the [data import
+chapter](https://r4ds.had.co.nz/data-import.html) in R for data science.
+
+The easiest way to install the latest version from CRAN is to install the whole tidyverse.
```

Comment:
Is that actually true?

Reply:
that is what we have always said for readr posts

Reply:
We probably should change it then; it won't actually upgrade, right?

Reply:
Yeah, not unless we bump the versions in the tidyverse package.